### ISSUE TITLE
Motivation and Mindset:  add more descriptive alt text

### ISSUE THREAD

**Issue by PridefulBlindford** [TheOdinProject/curriculum#31148]
### Checks

- [x] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [x] Would you like to work on this issue?

### Describe your suggestion

In the motivation section, when describing  how to get learners through an issue when stuck,there is an alt text for the image. However,this description is very short, and just says what the image is,but not the full contents of the image. There is a list,but it would be ehlpful to more fully describe the contents of the chart for those like me who can't see the image. 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-motivation-and-mindset

**Comment by mao-sz** [comment]
Thanks for opening this @ReadyPlayerN, we're aware that a lot of our image alt texts are in need of improvement, but it's overall quite  a lot of work for the whole curriculum which we will eventually get to, but other work has taken priority. What's helpful right now is that you're an AT user and could provide some practical insight for what helps most here.

The image is a diagram essentially showing the following flow:
1. Do lesson
1. Practise concept
1. Am I aware of how to solve it?
   1. Yes - End
   1. No - Have I researched and experimented?
      1. Yes - Have I asked for guidance?
         1. Yes - go to step 2 (practise concept)
         1. No - Ask a question. Does the guidance make any sense?
            1. Yes - go to step 2 (practise concept)
            1. No - go to step 1 (do lesson)
      1. No - Research and experiment then go to step 2 (practise concept)

For you, how would you prefer something like this to be presented? First thing I note is that a text version of the diagram does not exist i.e. the image is the only source of the info (not good). Would it help to have a text version of the diagram then associate that text with the image?

**Comment by PridefulBlindford** [comment]
So, since this seems to be a long description (which is good), I think it would be helpful to have this connected to the image, or say, have a link that goes to the long description as part of the iamge or near to it,since it is a detailed descritption of the flow chart. Having it connected to the image would also be good,as it would give a detailed explanation of what the image is displaying. But overall,yes, connecting the description to the image would be the best and good move.

**Comment by mao-sz** [comment]
We've been discussing the exact approach to take that will be sufficiently accessible while limiting how much it interferes with being able to write our lesson contents in Markdown. We've opted for now to go with [describing the location of the long description in the `alt` attribute](https://www.w3.org/WAI/tutorials/images/complex/#approach-2-describing-the-location-of-the-long-description-in-the-alt-attribute) and following the image with a headed subsection with the full text description of what the flowchart shows, similar to how I put it in my previous comment (or whatever format is equivalently easy to understand).

That'll require 2 things to be done:
1. Write the text version of the workflow directly before or after the image, and give it an appropriate level 4 heading.
1. Edit the image alt text so it says what heading the text version can be found under.

Is this something you still wish to be assigned to do @ReadyPlayerN?

**Comment by PridefulBlindford** [comment]
This seems like a pretty complex issues. I was mainly just flagging it to increase the accessibility overall of the oden project. I am glad that it seems pretty accessible, and that WSL is supported as a way to access the content. I don't think so.

**Comment by mao-sz** [comment]
No problem, I'll get this knocked out in due course. We have had discussions regarding curriculum-wide accessibility auditing but understandably, that's a very big task which would take a long time to fully complete, though we have gone through and improved accessibility of many parts over the years already. There will of course be things we still need to address (and we know that image alt text is one of them).

**Comment by BryanF1nes** [comment]
@mao-sz if you're unable to make time for this I'd be happy to assist as well.

**Comment by mao-sz** [comment]
Actually yeah, go for it

**Comment by BryanF1nes** [comment]
Just providing an update, I started working on this today and will try to have it finished within the next day or two.

**Comment by mao-sz** [comment]
@BryanF1nes checking in if this is something you're still wanting to work on

**Comment by BryanF1nes** [comment]
Hey @mao-sz thanks for checking in. I have been quite busy with some alternative things including family things, I am happy to work on this I just don't know when I will be able to get to it. If you or someone else has more free time please feel free to take this over for me! If not I can get to it hopefully within the next week or so!

**Comment by mao-sz** [comment]
I'll take this one then, since you also have https://github.com/TheOdinProject/curriculum/issues/28831
